src/main/scala/com/bsgenerator/extractor/article/HeuristicExtractor.scala

```diff
@@ -0,0 +1,87 @@
+package com.bsgenerator.extractor.article
+
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Element
+
+import scala.collection.mutable.Map
+import scala.collection.JavaConverters._
+import scala.collection.mutable
+
+object HeuristicExtractor {
+
+  val unlikelyClassCandidates: Array[String] = Array("-ad-", "ai2html", "banner", "breadcrumbs", "combx", "comment", "community", "cover-wrap", "disqus", "extra", "foot", "gdpr", "header", "legends", "menu", "related", "remark", "replies", "rss", "shoutbox", "sidebar", "skyscraper", "social", "sponsor", "supplemental", "ad-break", "agegate", "pagination", "pager", "popup", "yom-remote")
```

Comment:
If you've made these two collections fields of a companion object, they can't be edited anyway, am I right? So maybe they should be immutable sequences?
Here is an entry on scala sequences: https://docs.scala-lang.org/overviews/collections/overview.html

Reply:
You are correct, I have addressed the issue. New patch no longer calls toArray after asScala as well.